scripts/fetch_ext_bins.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+# Copyright 2018 The Kubernetes Authors.
```

Comment:
I'm not sure where this is from but I would prefer the version from the cluster-api repo. In this script here we delete a folder when SKIP_FETCH_TOOLS is set, which is a bit strange, as imho it shouldn't do anything.

Overall we try to stay consistent with the main cluster-api repo in CAPO to reduce maintenance effort on our side.

Reply:
The capi repo does not have the script. But, the ones in Amazon, which which I took this script and the one in GCP use the same approach. 
https://github.com/kubernetes-sigs/cluster-api-provider-gcp/blob/main/scripts/fetch_ext_bins.sh